core/blockchain.go

```diff
@@ -160,7 +160,7 @@ type CacheConfig struct {
 	SnapshotVerify                  bool          // Verify generated snapshots
 	Preimages                       bool          // Whether to store preimage of trie key to the disk
 	AcceptedCacheSize               int           // Depth of accepted headers cache and accepted logs cache at the accepted tip
-	TxLookupLimit                   uint64        // Number of recent blocks for which to maintain transaction lookup indices
+	TxLookupLimit                   int64         // Number of recent blocks for which to maintain transaction lookup indices
```

Comment:
Should we add a new parameter like`tx-indexing-disabled` instead of adding -1?
Seems this would simplify validation

Reply:
In that case we should validate it against TxLookupLimit and it might be confusing to use both of them at the same time.